robottelo/constants.py

```diff
@@ -48,10 +48,12 @@
 
 OPERATING_SYSTEMS = [
```

Comment:
Do you propose to add import statement to `constants.py` file?
It is pretty unusual move, but I can do that, just confirm that